distributed/stealing.py

```diff
@@ -313,16 +313,8 @@ async def move_task_confirm(self, *, key, state, stimulus_id, worker=None):
             _log_msg = [key, state, victim.address, thief.address, stimulus_id]
 
             if ts.state != "processing":
-                self.log(("not-processing", *_log_msg))
-                old_thief = thief.occupancy
-                new_thief = sum(thief.processing.values())
-                old_victim = victim.occupancy
-                new_victim = sum(victim.processing.values())
-                thief.occupancy = new_thief
-                victim.occupancy = new_victim
-                self.scheduler.total_occupancy += (
-                    new_thief - old_thief + new_victim - old_victim
-                )
+                self.scheduler._reevaluate_occupancy_worker(thief)
+                self.scheduler._reevaluate_occupancy_worker(victim)
```

Comment:
Since it looks like `_reevaluate_occupancy_worker` was removed (even as a method), should we be doing something else here?

Reply:
Ahh, I missed this one. I think this is why I put it in as a mathod to avoid cyclic imports

Reply:
FWIW did like the idea you had of getting rid of this function if that is an option. Though admittedly there might need to be more thought on this test

Reply:
Intuitively, it feels like this should not even be here since this interacts pretty deeply with the scheduler and the stealing thing _should_ be an extension and not interact on this deep level. However, the only way to not do this here would be to solely rely on the _eventual_ update via a callback. No idea how disruptive this would be